app/src/main/java/com/example/themoviesdb/ui/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.themoviesdb.ui
+
+import android.os.Bundle
+import android.view.*
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import com.bumptech.glide.Glide
+import com.example.themoviesdb.R
+import com.example.themoviesdb.databinding.FragmentMovieDetailsBinding
+import com.example.themoviesdb.domain.model.Movie
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MovieDetailsFragment : Fragment() {
+
+    private lateinit var binding: FragmentMovieDetailsBinding
+    private var movie: Movie? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        movie = arguments?.getParcelable("movie")
+        binding = DataBindingUtil.inflate<FragmentMovieDetailsBinding>(
+            inflater,
+            R.layout.fragment_movie_details,
+            container,
+            false
+        ).apply {
+            lifecycleOwner = viewLifecycleOwner
+        }
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initUI()
+    }
+
+    private fun initUI() {
+        (activity as MainActivity).supportActionBar?.title = movie?.title
+        with(binding){
```

Comment:
the block below "with(binding) { ... }" can be replaced by data binding in layout xml.